source-mysql/main.go

```diff
@@ -218,6 +218,11 @@ func (db *mysqlDatabase) Connect(ctx context.Context) error {
 		return fmt.Errorf("unable to connect to database: %w", err)
 	}
 
+	// Set our desired timezone to UTC. This is required for backfills of timestamp columns to behave consistently.
+	if _, err := db.conn.Execute("SET time_zone = '+00:00';"); err != nil {
```

Comment:
I was originally wondering if this changed the time zone of the database itself, but apparently its actually a per session setting, so that's good.